src/VisualStudio/VisualStudioInteractiveComponents/AssemblyRedirects.cs

```diff
@@ -5,17 +5,17 @@
 
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.Scripting.dll")]
 [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.CSharp.Scripting.dll")]
-[assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
+// [assembly: ProvideRoslynBindingRedirection("Microsoft.CodeAnalysis.VisualBasic.Scripting.dll")]
```

Comment:
This is the interactive window redirects only.  VB is never loaded in the interactive window.  @tmat verified we could remove this redirect.  Not sure what scenario you're worried about here.